src/feature-selector-form/feature-selector-form.module.css

```diff
@@ -147,6 +144,23 @@ button.clearButton:active {
 	max-width: 20rem;
 }
 
+.selectedFeatureKeywordTitle {
+	font-size: var( --font-body );
+	font-weight: 500;
+	margin-bottom: 0.375rem;
+}
+
+.keywordsWrapper {
+	display: flex;
+	flex-wrap: wrap;
+}
+
+.noKeywords {
+	font-style: italic;
+	color: var( --color-error );
+	font-size: var( --font-header );
```

Comment:
> NOTE: I used the same style for the bad tasks. I'm open to updating the style if needed! (cc: @john-legg )

**nitpick (blocking):** Ah, yeah I think the style should be updated to not use the bad task style. This is mostly so the user doesn't think they did something wrong or there's something wrong with the reporting config. I apologize for not providing a style guide for this edge case! I would suggest keeping the style similar to pills for consistency:

```
.noKeywords {
    color: var(--color-text-primary);
    font-size: var(--font-body-small);
}
```

Reply:
Ah, that makes sense! The initial design looked like there was an issue with the keywords. I've update the CSS here: [63435cc](https://github.com/Automattic/bugomattic/pull/77/commits/63435cc6fda30b3a125afaae25cfd02e001e3c9e) and now it should look like this

![Screenshot 2023-04-21 at 11 47 37](https://user-images.githubusercontent.com/67279475/233605082-81f755a7-5485-400e-a6fc-4c556ef2cd7b.png)